cpp/src/groupby/common/aggregation_requests.hpp

```diff
@@ -52,6 +53,40 @@ using AggRequestType = std::pair<gdf_column*, operators>;
  *---------------------------------------------------------------------------**/
 using SimpleAggRequestCounter = std::pair<AggRequestType, gdf_size_type>;
 
+static constexpr std::array<operators, 4> simple_aggregations = {SUM, MIN, MAX, COUNT};
+
+static constexpr std::array<operators, 2> ordered_aggregations = {MEDIAN, QUANTILE};
+
+// Just an utility function to find the existence of on element in a constexpr array
+template <class T, size_t N>
+constexpr bool array_contains(std::array<T,N> const& haystack, T needle){
+    for(auto i = 0u; i < N; ++i){
+       if(haystack[i] == needle)
+           return true;
+    }
+    return false;
+}
+
+/**---------------------------------------------------------------------------*
+ * @brief  To verify that the input operator is part of simple_aggregations list.
+ * Note that in this kind of aggregators can be computed in a single pass scan.
+ * In the other hand the compound aggregation MEAN need to be computed by simple
+ * ones (SUM and COUNT).
+ *---------------------------------------------------------------------------**/
+inline bool is_simple(operators op) {
+  return array_contains(simple_aggregations, op);
+}
+
+
+/**---------------------------------------------------------------------------*
+ * @brief  To verify that the input operator is part of  ordered_aggregations list.
+ * Ordered aggregation is used to identify other ones like MEDIAN and  QUANTILE,
+ * which cannot be represented as a combination of single-pass aggregations. 
+ *---------------------------------------------------------------------------**/
+inline bool is_ordered(operators op) {
```

Comment:
```suggestion
constexpr inline bool is_ordered(operators op) {
```

Reply:
If I declare `is_ordered` and `is_simple` as constexpr, I get an compilation error. 
```
Error: Internal Compiler Error (codegen): "there was an error in verifying the lgenfe output!"
```

Reply:
Hm, I guess we'll just sort that out later when we actually need these to be `constexpr`.